src/date.field-widget.js

```diff
@@ -64,7 +64,7 @@ function date_field_widget_form(form, form_state, field, instance, langcode, ite
     }
     if (!value2_set && items[delta].default_value2 != '') {
       switch (items[delta].default_value2) {
-        case 'same':
+        case 'now':
```

Comment:
@mkinnan Good catch. I think we'll want to cover both of them here:

```
case 'same':
case 'now':
```

Thoughts?